tests/functional/test_freeze.py

```diff
@@ -258,41 +259,31 @@ def test_freeze_mercurial_clone(script, tmpdir):
     _check_output(result, expected)
 
 
-@pytest.mark.network
 def test_freeze_bazaar_clone(script, tmpdir):
     """
     Test freezing a Bazaar clone.
 
     """
-
-    checkout_path = local_checkout(
-        'bzr+http://bazaar.launchpad.net/%7Edjango-wikiapp/django-wikiapp/'
-        'release-0.1',
-        tmpdir.join("cache"),
-    )
-    # bzr internally stores windows drives as uppercase; we'll match that.
+    try:
+        checkout_path = _create_test_package(script, vcs='bazaar')
+    except OSError as e:
+        pytest.skip('Invoking `bzr` failed: %s' % e)
     checkout_pathC = checkout_path.replace('c:', 'C:')
 
     result = script.run(
-        'bzr', 'checkout', '-r', '174',
-        local_repo(
-            'bzr+http://bazaar.launchpad.net/%7Edjango-wikiapp/django-wikiapp/'
-            'release-0.1',
-            tmpdir.join("cache"),
-        ),
-        'django-wikiapp',
+        'bzr', 'checkout', checkout_path, 'bzr-package'
     )
     result = script.run(
         'python', 'setup.py', 'develop',
-        cwd=script.scratch_path / 'django-wikiapp',
+        cwd=script.scratch_path / 'bzr-package',
         expect_stderr=True,
     )
     result = script.pip('freeze', expect_stderr=True)
     expected = textwrap.dedent("""\
         Script result: ...pip freeze
         -- stdout: --------------------
-        ...-e %s@...#egg=django_wikiapp-...
-        ...""" % checkout_pathC)
+        ...-e bzr+file://%s@1#egg=version_pkg-0.1-...
+        ...""" % checkout_path)
```

Comment:
I'd have to fire up a Windows VM and probably install bzr in said VM to test this. Ideally, we can find someone who has both Windows and bzr and enjoys doing Python coding and debugging on Windows. I'm much more comfortable with POSIX myself, so I'd probably fumble my way through the Windows issues.

Oh but then actually you said that there are other issues with running the tests on Windows -- so maybe it's not worth worrying too much about how this test works in Windows until we've fixed the overall problem with running tests on Windows.

Otherwise, I fear we will get stuck on Windows issues and the PR will stagnate and we won't get the benefit of having the Travis CI tests not intermittently fail and confuse people.

I think the tests **should** work on Windows; I just think it's a bigger task than this PR and so I'm inclined to treat it separately.